payments_service/braintree/views.py

```diff
@@ -251,36 +266,47 @@ def notify_buyer(self, result):
                 .format(notice_kind)
             )
 
-        tpl = get_template('braintree/emails/{}.txt'.format(notice_kind))
-        text_body = tpl.render(Context(dict(
-            moz_trans=moz_trans,
-            product=product,
-            seller=sellers_by_product[product.id],
-            result=result,
-            date=parse_date(moz_trans['created']).strftime('%d %b %Y'),
-            transaction=moz_trans,
-            cc_type=result['mozilla']['paymethod']['type_name'],
-            cc_truncated_id=result['mozilla']['paymethod']['truncated_id'],
-            bill_start=parse_date(
-                bt_trans['billing_period_start_date']).strftime('%d %b %Y'),
-            bill_end=parse_date(
-                bt_trans['billing_period_end_date']).strftime('%d %b %Y'),
-            next_pay_date=parse_date(
-                bt_trans['next_billing_date']).strftime('%d %b %Y'),
-        )))
-
+        data = self.build_context(
+            bt_trans, moz_trans, result['mozilla']['paymethod'], product)
         connection = get_connection(fail_silently=False)
+
         mail = EmailMultiAlternatives(
             subject,
-            text_body,
+            self.render(data, notice_kind, 'txt'),
             settings.SUBSCRIPTION_FROM_EMAIL,
             [result['mozilla']['buyer']['email']],
             reply_to=[settings.SUBSCRIPTION_REPLY_TO_EMAIL],
             connection=connection)
 
-        # TODO: send an HTML email.
-        # https://github.com/mozilla/payments-service/issues/59
-        #
-        # mail.attach_alternative(html_message, 'text/html')
-
+        mail.attach_alternative(
+            self.render(data, notice_kind, 'html'),
+            'text/html')
         mail.send()
+
+
+def webhook_email(request):
+    if not settings.DEBUG:
```

Comment:
This is intended as an easy way to see the email and play with it in developer tools.